pkg/controller/admissionchecks/multikueue/workload.go

```diff
@@ -276,7 +277,7 @@ func (a *wlReconciler) reconcileGroup(ctx context.Context, group *wlGroup) error
 			Type:    kueue.WorkloadFinished,
 			Status:  metav1.ConditionTrue,
 			Reason:  remoteFinishedCond.Reason,
-			Message: fmt.Sprintf("From remote %q: %s", remote, remoteFinishedCond.Message),
```

Comment:
Why changing this? it could be useful for users to know immediately which worker is processing the job.

Reply:
> The message prefix has very little added value since the remote used will remain recorded in the admission check state.

sgtm, in that case I think we could drop the prefix indeed in that case. However, I would like to make to ensure there is a way users can get to know the worker doing computations. Do we have an assert for this in e2e tests? Maybe extend this test to show the information is surfaced.

Reply:
I see, it is here in e2e tests: https://github.com/kubernetes-sigs/kueue/blob/a8c11f351858cd159a3f823b15860918c0490cbc/test/e2e/multikueue/e2e_test.go#L208

In that case, I'm ok to revmove it from the condition message.